File: solutions/Minimum sum four digits after splitting digits.cpp
class Solution {
public:
    int minimumSum(int num) {
        vector<int> dig;
        while(num>0){
            dig.push_back(num%10);
            num/=10;
        }
        sort(dig.begin(), dig.end());
        return (dig[0]*10+dig[3])+(dig[1]*10+dig[2]);
    }
};

File: solutions/Minimum sum four digits after splitting digits.py
class Solution:
    def minimumSum(self, num: int) -> int:
        dig = []
        while(num>0):
            dig.append(num%10)
            num//=10
        dig.sort()
        return (dig[0]*10+dig[3])+(dig[1]*10+dig[2])